src/test/TestUtils/TestUtils.java

```diff
@@ -808,6 +815,35 @@ public static JSONObject getFieldValues(InputStream inputStream) throws IOExcept
     return fieldValues;
   }
 
+  public static void assertPDFEquals(InputStream inputStream1, InputStream inputStream2) {
+    try {
+      PDDocument pdfDocument1 = Loader.loadPDF(inputStream1);
+      PDDocument pdfDocument2 = Loader.loadPDF(inputStream2);
+
+      PDPageTree pdfDocumentPages1 = pdfDocument1.getPages();
+      PDPageTree pdfDocumentPages2 = pdfDocument2.getPages();
+      Iterator<PDPage> pdfDocumentIterator1 = pdfDocumentPages1.iterator();
+      Iterator<PDPage> pdfDocumentIterator2 = pdfDocumentPages2.iterator();
+
+      while (pdfDocumentIterator1.hasNext() && pdfDocumentIterator2.hasNext()) {
+        Iterator<PDStream> contentIterator1 = pdfDocumentIterator1.next().getContentStreams();
+        Iterator<PDStream> contentIterator2 = pdfDocumentIterator2.next().getContentStreams();
+
+        while (contentIterator1.hasNext() && contentIterator2.hasNext()) {
+          assertArrayEquals(contentIterator1.next().toByteArray(), contentIterator2.next().toByteArray());
```

Comment:
does this literally just iterate through the entire byte array of the file? Shouldn't there be a better way to check like just calculating a hash